include/exadg/time_integration/time_integration_constants_base.h

```diff
@@ -70,6 +71,19 @@ class TimeIntegratorConstantsBase
   print(dealii::ConditionalOStream & pcout) const = 0;
 
 protected:
+  /**
+   * Can be used to fill the unused components of the vector with zeros.
```

Comment:
Here, one could add the information that the last entries of the vector are zeroed out corresponding to time instants lagging behind most.

Reply:
[a63b519](https://github.com/exadg/exadg/pull/469/commits/a63b51958000e5041f91ccbb314b126ec95e8b48)